src/Tgstation.Server.Host/Components/InstanceManager.cs

```diff
@@ -675,6 +675,11 @@ void CheckSystemCompatibility()
 			{
 				if (!systemIdentity.CanCreateSymlinks)
 					throw new InvalidOperationException($"The user running {Constants.CanonicalPackageName} cannot create symlinks! Please try running as an administrative user!");
+
+				if (!platformIdentifier.IsWindows && systemIdentity.IsSuperUser)
+				{
+					logger.LogWarning("TGS is being run as the root account. This is not recommended and may prevent launch in a future version.");
```

Comment:
This is never really gonna happen
```suggestion
					logger.LogWarning("TGS is being run as the root account. This is not recommended.");
```